heron/stmgr/src/cpp/manager/stmgr.cpp

```diff
@@ -244,12 +250,13 @@ void StMgr::CheckTMasterLocation(EventLoop::Status) {
   }
 }
 
-void StMgr::UpdateProcessMetrics(EventLoop::Status) {
-  // Uptime
+void StMgr::UpdateUptimeMetric() {
```

Comment:
UpdateUptimeMetric() is called 60 times every min, UpdateProcessMetrics() is called every min.
is it overlap 1 in every 60 times?

Reply:
yeah. that is right.